nlp/compat.js

```diff
@@ -120,10 +120,22 @@ function deviceNames(locale, result, entities) {
     result.code = newCode;
 }
 
+function defaultUnits(locale, result, entitites) {
+    for (let i = 0; i < result.code.length; i++) {
+        if (result.code[i].startsWith('default')) {
```

Comment:
This will run in NN syntax, so the token will be `unit:defaultTemperature`, and it should be replaced with `unit:F`.

Reply:
Fixed!